### PR TITLE
(WIN-73) Finish Win-2008r2 Appveyor packer image preparation 

### PR DESCRIPTION
**Pull request by jcoconnor**
Continue GCE Builds for Appveyor to bring image up to a ready state that it can be used to run Appveyer PR tests against it.

This involves:
1. Standardising Build names to match existing packer/puppet build practice.
2. Creating appveyor user account.
3. Installing the appveyor environment, including 3rd party software and appveyor agent.
4. Provisioning the appveyor user to auto login and run the appveyor base agent.

### CODE REVIEW

File: templates/win/common/scripts/google/install-appveyor-packages.ps1 [hunk 1]
```diff
@@ -47,6 +47,7 @@ Run-AppveyorProvisionScript -ScriptName  $PackerScriptsDir\enterprise\add_ssh_kn
 Run-AppveyorProvisionScript -ScriptName  $PackerScriptsDir\enterprise\install_appveyor_build_agent.ps1 -Description 'Install Appveyor Build Agent'
 
 Run-AppveyorProvisionScript -ScriptName  $PackerScriptsDir\enterprise\set_gce_build_agent_mode.ps1 -Description 'Set GCE Build Agent Mode'
+Run-AppveyorProvisionScript -ScriptName  $PackerScriptsDir\enterprise\add_appveyor_build_agent_to_auto_run.ps1 -Description 'Set Set AutoRun Script'
```
Comment (RandomNoun7) [inline templates/win/common/scripts/google/install-appveyor-packages.ps1]: Is "Set Set" correct?

File: templates/win/common/scripts/google/install-appveyor-packages.ps1 [hunk 2]
```diff
@@ -70,3 +70,4 @@ autologon "$appveyor_username" . "$appveyor_password"
 
 Start-Process -Wait -Credential $Credential -LoadUserProfile -FilePath "Powershell.exe" -ArgumentList ("-file $PackerScriptsDir\enterprise\set_gce_build_agent_mode.ps1")
 
+Write-Output "Appveyor Apps Installation Completed"
```
Comment (RandomNoun7) [inline templates/win/common/scripts/google/install-appveyor-packages.ps1]: Line break.

File: templates/win/common/scripts/google/install-appveyor-packages.ps1 (hunk 1 above)
Comment (jcoconnor) [reply templates/win/common/scripts/google/install-appveyor-packages.ps1]: Indeed it is not - to be fixed

**Comment by jcoconnor** [comment]
This PR is ready for review - last commit added Ruby which means image should be ready to perform PR validation.

Running it through manual build and deploy to appveyor environment to test against a bolt build.

**Comment by jcoconnor** [comment]
Not quite ready for merge yet - MSYS2 is missing and possibly some other tools.

**Comment by jcoconnor** [comment]
Ok - Marking this as ready for Merge now.
I have been able to get a successful test run through for the [puppetlabs-powershell Repo](https://github.com/puppetlabs/puppetlabs-powershell) - [see PR 201](https://github.com/puppetlabs/puppetlabs-powershell/pull/210)

There are still issues with the Bolt repo:
1. Test Harness assumes WMF 5 features - ticket [BOLT-342](https://tickets.puppetlabs.com/browse/BOLT-341)
2. MSYS and/or some other packages may also be needed on the box - will deal with this in separate ticket (TBD)

**Comment by jcoconnor** [comment]
Test run for [puppetlabs-powershell Repo](https://github.com/puppetlabs/puppetlabs-powershell) available at: https://ci.appveyor.com/project/puppetlabs/puppetlabs-powershell/build/1.1.x.733

**Pull request merged**